### PR TITLE
Custom CAs in operator

### PR DESCRIPTION
**Pull request by mkaesz-wandb**
## Description

How to configure Custom CAs in operator.

## Ticket

Does this PR fix an existing issue? If yes, provide a link to the ticket here:

## Checklist

Check if your PR fulfills the following requirements. Put an `X` in the boxes that apply.

- [ x] Files I edited were previewed on my local development server with `yarn start`. My changes did not break the local preview.
- [x ] Build (`yarn docusaurus build`) was run locally and successfully without errors or warnings.
- [ x] I merged the latest changes from `main` into my feature branch before submitting this PR.

### CODE REVIEW

**Review by flamarion — COMMENTED** [review]
Not directly connected with the Custom CA, but I wonder if we really [this](https://github.com/wandb/docodile/blob/custom-ca/docs/guides/hosting/operator.md?plain=1#L440-L441) or if this is only an example.
If it's an example, I suggest we use a different env var, this can be confused. In the end, when setting up the bucket, we already set the `region` option that will become that same env var, so adding it in a different place will override what we're adding for the bucket, or vice versa.

**Review by flamarion — APPROVED** [review]
LGTM 🚢

File: docs/guides/hosting/operator.md [hunk 1]
```diff
@@ -705,6 +703,60 @@ global:
...
+    SAAwRQIhAIzXZMW44l6XMf9Nf4TxTevK8vE4Ic6E8UFqsCcILdXjAiA7iTluM0IU
+    aIgJYVqKxXt25blH/VyBRzvNhViesfkNUQ==
+    -----END CERTIFICATE-----
+```
+
+## Configuration Reference (W&B Operator)
+
+This section describes the configuration options for W&B Kubernetes operator (wandb-controller-manager). The operator receives its configuration in the form of a YAML file. 
+
+By default, the W&B Kubernetes operator does not need any configuration file and should only be created if actually needed.
```
Comment (ngrayluna) [inline docs/guides/hosting/operator.md]: @mkaesz-wandb  What is an example or two if when someone would need to create a config file?

**Review by ngrayluna — COMMENTED** [review]
Left one QQ.

File: docs/guides/hosting/operator.md (hunk 1 above)
Comment (mkaesz-wandb) [reply docs/guides/hosting/operator.md]: Added two examples. Thanks!

**Pull request merged**